Apps/JobScheduler/src/Jobs/NotificationSettingsJob.cs

```diff
@@ -74,51 +74,47 @@ public class NotificationSettingsJob : INotificationSettingsJob
         public void PushNotificationSettings(string notificationSettingsJson)
         {
             this.logger.LogDebug("Queueing Notification Settings push to PHSA...");
-            if (this.jobEnabled)
+            if (!this.jobEnabled)
             {
-                NotificationSettingsRequest? notificationSettings = JsonSerializer.Deserialize<NotificationSettingsRequest>(notificationSettingsJson);
-                if (notificationSettings != null)
-                {
-                    string? accessToken = this.authDelegate.AuthenticateAsSystem(this.tokenUri, this.tokenRequest).AccessToken;
+                this.logger.LogInformation("Job has been disabled by configuration");
+                return;
+            }
 
-                    if (string.IsNullOrEmpty(accessToken))
-                    {
-                        this.logger.LogError("Authenticated as User System access token is null or empty, Error:\n{AccessToken}", accessToken);
-                        throw new FormatException($"Authenticated as User System access token is null or empty, Error:\n{accessToken}");
-                    }
+            NotificationSettingsRequest? notificationSettings = JsonSerializer.Deserialize<NotificationSettingsRequest>(notificationSettingsJson);
+            if (notificationSettings == null)
+            {
+                this.logger.LogError("Unable to deserialize JSON Notification Settings");
+                throw new FormatException("Unable to deserialize JSON Notification Settings");
```

Comment:
Not fond of using FormatException here - InvalidOperationException might be more sensible, however, it was there before :)